conda/cli/common.py

```diff
@@ -575,3 +577,21 @@ def disp_env(prefix):
 
     if output:
         print()
+
+
+DEPRECATED = ["force_pscheck", ]
+
+
+def deprecation_warning(func):
+    """Wraps an execute function in a deprecated args checker"""
+
+    msg = "Argument %s is no longer used"
+
+    @wraps(func)
+    def inner(args, parser):
+        for key in DEPRECATED:
+            if not hasattr(args, key):
+                continue
+            warnings.warn(msg % key.replace("_", "-"), DeprecationWarning)
```

Comment:
Anyway, warnings for CLI interfaces should probably just be a print to stderr. `warnings.warn` is not very user-friendly in terms of its output.